bo/avis.go

```diff
@@ -68,12 +68,12 @@ func AvisStructLevelValidationAbweichung(sl validator.StructLevel) {
 	avis := sl.Current().Interface().(Avis)
 	declineAvis := avis.AvisTyp == avistyp.ABGELEHNTE_FORDERUNG
 	for _, avisPosition := range avis.AvisPositionen {
-		if declineAvis && (avisPosition.Abweichung == nil || avisPosition.Abweichung.AbweichungsGrund == nil || *avisPosition.Abweichung.AbweichungsGrund == 0) {
-			sl.ReportError(avisPosition.Abweichung, "AbweichungsGrund", "Abweichung", "Avis.AvisPositionen.Abweichung->AbgelehnteForderungRequired", "")
+		if declineAvis && (avisPosition.Abweichungen == nil || avisPosition.Abweichungen[0].AbweichungsGrund == nil || *avisPosition.Abweichungen[0].AbweichungsGrund == 0) {
```

Comment:
```suggestion
		if declineAvis && (len(avisPosition.Abweichungen) == 0 || avisPosition.Abweichungen[0].AbweichungsGrund == nil || *avisPosition.Abweichungen[0].AbweichungsGrund == 0) {
```
das hat @hf-kklein gut gesehen